build-scripts/combine_remediations.py

```diff
@@ -144,11 +144,11 @@ def main():
                             remediation_functions
                         )
                 else:
-                    sys.stderr.write("Skipping '%s' remediation script. "
+                    sys.stderr.write("Warning: Skipping '%s' remediation script. "
                                      "The platform identifier in the "
                                      "script is missing!\n" % (filename))
-    sys.stderr.write("Merged %d %s remediations.\n"
-                     % (included_fixes_count, args.remediation_type))
+    #sys.stderr.write("Merged %d %s remediations.\n"
+    #                 % (included_fixes_count, args.remediation_type))
```

Comment:
I think we should keep this, it confirms that stuff is working and the output is not overwhelming.